Documentation/gettingstarted/istio.rst

```diff
@@ -145,10 +145,13 @@ Step 3: Deploy the Bookinfo Application V1
 
 Now that we have Cilium and Istio deployed, we can deploy version
 ``v1`` of the services of the `Istio Bookinfo sample application
-<https://istio.io/docs/guides/bookinfo.html>`_.
+<https://istio.io/docs/examples/bookinfo.html>`_.
 
-The BookInfo application is broken into four separate microservices:
+While the upstream `Istio Bookinfo
```

Comment:
This line returns too early, fix the formatting.

Reply:
Done.